src/ducks/devtools/Queries.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useMemo } from 'react'
+import sortBy from 'lodash/sortBy'
+import Grid from '@material-ui/core/Grid'
+import Box from '@material-ui/core/Box'
+
+import { useClient } from 'cozy-client'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import ListItem from 'cozy-ui/transpiled/react/ListItem'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import { useSelector } from 'react-redux'
+import format from 'date-fns/format'
+import Inspector from 'react-inspector'
+
+import { NavSecondaryAction, ListGridItem } from './common'
+
+const styles = {
+  panelRight: { height: '100%', overflow: 'scroll', flexGrow: 1 },
+  mono: { fontFamily: 'monospace' }
+}
+
+const FetchStatus = ({ fetchStatus }) => {
+  return (
+    <span
+      className={
+        fetchStatus == 'loaded'
+          ? 'u-valid'
+          : fetchStatus === 'pending'
+          ? 'u-warn'
+          : fetchStatus === 'error'
+          ? 'u-error'
+          : ''
+      }
+    >
+      {fetchStatus}
+    </span>
+  )
+}
+
+const QueryData = ({ data, doctype }) => {
+  const client = useClient()
+
+  const documents = useSelector(state => state.cozy.documents[doctype])
+
+  const storeData = useMemo(() => {
+    return data.map(id => client.hydrateDocument(documents[id]))
+  }, [client, data, documents])
+  return <Inspector data={storeData} />
+}
+
+const QueryState = ({ name }) => {
+  const queryState = useSelector(state => state.cozy.queries[name])
+  const { data } = queryState
+  const { lastFetch, lastUpdate } = useMemo(() => {
+    return {
+      lastFetch: new Date(queryState.lastFetch),
+      lastUpdate: new Date(queryState.lastUpdate)
+    }
+  }, [queryState])
+  return (
+    <>
+      <table style={styles.mono}>
+        <tr>
+          <td>doctype</td>
+          <td>{queryState.definition.doctype}</td>
+        </tr>
+        <tr>
+          <td>fetchStatus</td>
+          <td>
+            <FetchStatus fetchStatus={queryState.fetchStatus} />
+          </td>
+        </tr>
+        <tr>
+          <td>lastFetch</td>
+          <td>{format(lastFetch, 'HH:mm:ss')}</td>
```

Comment:
Hmm, should we use f() from useI18n() or should we use the default format? Does the timezone is useful here?

Reply:
Since we do not internationalize anything, f() will not bring anything to the table. The timezone is not important here since we display really recent events in the same timezone as the user.